activestorage/active.py

```diff
@@ -222,43 +248,51 @@ def _from_storage(self, stripped_indexer, drop_axes, out_shape, out_dtype,
                 if self._method == "mean":
                     # For the average, the returned component is
                     # "sum", not "mean"
-                    out = {"sum": out, "n": n}
+                    out = {"sum": out, "n": sum(counts)}
```

Comment:
Further to my comments over at `storage.py`, instead of `out = method(out)` a few lines above, we'd have
```python
out = np.array(out)[np.where(counts >= 0)]  # Throw away missing values in 'out'
out = method(out)
```